shopinvader_image_search_engine_proxy/models/se_backend.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion (http://www.akretion.com).
+# Copyright 2018 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import urlparse
+from odoo import api, fields, models
+
+
+class SeBackend(models.Model):
+    _inherit = 'se.backend'
+
+    proxy_url = fields.Char(
+        help="Replace the original url (base part) by this proxy url during "
```

Comment:
@acsonefho The `help` is not right. The process doesn't apply magically to all the fields. Moreover I've to admit that I'm not comfortable with the field name `proxy_url` (website_url, image_proxy_url, ???)

Reply:
@acsonefho It's not possible to put this information on the locomotive_backend. I would prefer to have all the information about the website on the locomotive backend